File: public/phase-2-20160620022203/C_PLUS_PLUS/elegance-of-a-sequence.cpp
// Name : Elegance of a Sequence
#include <iostream>
#include <string>
using namespace std;

int main(int argc, char const *argv[]) {
	// input:
	// n : length of sequence of a
	// a : sequence a
	int index;
	int n;
	cin >> n;
	int a[105];

	for(index=0;index<n;index++)
		cin >> a[index];


	// write your code here
	// store your results in `k`,`b`

	// output
	// Dummy Data
	int k=2, b[105]={4,5};

	cout << k << endl;
	for(index=0;index<k;index++)
		cout << b[index] << " ";

}

File: public/phase-2-20160620022203/C_SHARP/elegance-of-a-sequence.cs
// Name : Elegance of a Sequence
using System;
namespace Application {
	class eleganceofa_sequence {
		public static void Main(string[] args) {
		// input:
		// n : length of sequence of a
		// a : sequence a
		int index;
		int n = Convert.ToInt32(Console.ReadLine());
		int[] a = new int[n];
		string[] a_elements = (Console.ReadLine()).Split(' ');
		for(index=0;index<n;index++)
			a[index] = Convert.ToInt32(a_elements[index]);


		// write your code here
		// store your results in `k`,`b`

		// output
		// Dummy Data
		int k=2;
		int[] b = new int[] {4,5};

		Console.WriteLine(k);
		Console.WriteLine(string.Join(" ", b));
		}	
	}
}